### ISSUE TITLE
configurable pathes for fligtperformance data

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#77]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/77) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

We need to make the pathes to setup performance files configurable from within the mss_settings.json

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

proposed solution

```
#!patch
diff --git a/mslib/performance/dlraircraft.py b/mslib/performance/dlraircraft.py
index 907e85c..a1c6bc1 100644
--- a/mslib/performance/dlraircraft.py
+++ b/mslib/performance/dlraircraft.py
@@ -39,6 +39,7 @@ try:
 except:
     modulepath = ""
 
+from mslib.msui import wms_login_cache
 """
 HALO
 """
@@ -61,7 +62,7 @@ class HALO(FliteStarExportedAircraft):
         """Defines the performance file (exported by FliteStar) and which file
         sections map to which flight mode.
         """
-        super(HALO, self).__init__(os.path.join(modulepath, "data", "HALO_+10DC.txt"))
+        super(HALO, self).__init__(os.path.join(wms_login_cache.DEFAULT_CONFIG_PATH, "data", "HALO_+10DC.txt"))
 
         # Define a mapping from the sections in the FliteStar file to the
         # individual flight modes. Keys are:
@@ -233,7 +234,7 @@ class Falcon(FliteStarExportedAircraft):
     def __init__(self):
         """
         """
-        super(Falcon, self).__init__(os.path.join(modulepath, "data", "FALCON_Baseline.txt"))
+        super(Falcon, self).__init__(os.path.join(wms_login_cache.DEFAULT_CONFIG_PATH, "data", "FALCON_Baseline.txt"))
 
         # Define a mapping from the sections in the FliteStar file to the
         # individual flight modes. Keys are:


```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

@wxmetvis any comments on that patch ^^

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

default path for performance data files set to DEFAULT_CONFIG_PATH, fixes #77
